rust/operator-binary/src/stackable-nifi-operator.rs

```diff
@@ -49,17 +47,5 @@ async fn main() -> Result<(), error::Error> {
 
     let client = client::create_client(Some("nifi.stackable.tech".to_string())).await?;
 
-    if let Err(error) = stackable_operator::crd::wait_until_crds_present(
```

Comment:
Why did you move this to the `create_controller` function?
I'm not really opposed, happy to merge as is, just wondering.

Reply:
I would required the kube-rs and k8s-openapi dependency in the binary as well.